torchbenchmark/models/demucs/demucs/model.py

```diff
@@ -64,19 +66,19 @@ class Demucs(th.jit.ScriptModule):
 
     @capture_init
     def __init__(self,
-                 sources=4,
-                 audio_channels=2,
-                 channels=64,
-                 depth=6,
-                 rewrite=True,
-                 glu=True,
-                 upsample=False,
-                 rescale=0.1,
-                 kernel_size=8,
-                 stride=4,
-                 growth=2.,
-                 lstm_layers=2,
-                 context=3):
+                 sources: int=4,
+                 audio_channels: int=2,
+                 channels: int=64,
+                 depth: int=6,
+                 rewrite: bool=True,
+                 glu: bool=True,
+                 upsample: bool=False,
+                 rescale: float=0.1,
+                 kernel_size: int=8,
+                 stride: int=4,
+                 growth: float=2.,
+                 lstm_layers: int=2,
+                 context: int=3) -> None:
```

Comment:
One usability improvement thought: if a parameter is unannotated but has a non-`None` default value, we can deduce parameter types based on the default values. I think this would be strictly better than assuming the default `Tensor` type. For default value of `None`, it is better to assume its type to be `Optional[Tensor]` than the default `Tensor` type.